migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,10 @@
+var PrisonersDilemma = artifacts.require("./PrisonersDilemma.sol");
+
+module.exports = function(deployer, network, accounts) {
+    
```

Comment:
Probably should add conditional code that changes depending on the network being used.